isarojdahal/learn.everydaykarmaa.com
# added javasript youtube playlist documentation

Description: This PR adds individual documentation pages for each video in the JavaScript YouTube playlist. Each page contains an embedded video and a link to the source code, providing a simple and focused learning experience.

**Changes Made:**
- Created a directory structure for the JavaScript playlist within the `notes` folder.
- Added individual Markdown files for each video, embedding the video directly from YouTube.
- Included a source code link on each page for users to access the related codebase.

**Why This PR Is Needed:**
- Organizes the video content into easily accessible individual pages.
- Provides learners with direct access to videos and related code, enhancing the learning experience.

**How to Test:**
- Navigate to the `notes/javascript` directory.
- Open any of the video pages and verify that the video is correctly embedded.
- Click the source code link to ensure it points to the correct repository or file.

<img width="941" alt="image" src="https://github.com/user-attachments/assets/dc32b39f-4fcf-456c-bb3d-2cf9ca239099">

Related Issues: Closes #10 from everydaykarmaa.com (if applicable)
